### ISSUE TITLE
Reference page numbers

### ISSUE THREAD

**Issue by ecurrano** [paleobot/pbot-dev#261]
Page numbers on references are not always numeric, and I got an error when I tried to enter a combination of letters and numbers:
![Screen Shot 2024-02-05 at 2 39 50 PM](https://github.com/paleobot/pbot-dev/assets/72828382/1067a4ae-1fa0-417e-9b5e-cc0ef4187e71)

**Comment by aazaff** [comment]
Oh interesting. I forgot now that it is common for page numbers to not be always numeric. Before we implement this, would we be okay with just not having page numbers? I forget if it is required by PBDB. Bouncing back to @ecurrano

**Comment by ecurrano** [comment]
Confirmed by Mark: page numbers are not required by PBDB. 

I also asked Mark about online only papers and how these page numbers should be entered into PBDB, and he said:
"Make something up. We have no specific protocol for this situation, but we are in the process of re-doing this form, so we will definitely cover this base soon." So, our options are: 1) encourage people to enter the DOI, which has the article number. 2) Have people enter this in the "number" field. 3) Have people enter it in the notes. Personally, I like #2. None of these options require any changes to PBot, just for the how-to guides. @doricon : thoughts?

**Comment by doricon** [comment]
I agree that the "number" field is a good place for it. Can we still allow the page number fields to be alpha-numeric since that could possibly arise for other reasons too, and just to future proof it?
